src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/RabbitMq/RabbitMqHelper.cs

```diff
@@ -195,12 +209,13 @@ public static string GetServerAddress(InstrumentedMethodCall instrumentedMethodC
             }
             catch
             {
+                agent.Logger.Warn("Unable to get RabbitMQ server address/port due to differences in the expected types. Server address/port attributes will not be available.");
```

Comment:
Could we pass in the exception to these, too? That should make it easier to fix if we run into an issue in the wild

Reply:
Done!